README.md

```diff
@@ -23,32 +23,7 @@ installed. This helps with making the builds reproducible and the provenances
 verifiable. The toml file should conform to the `BuildConfig` structure defined
 in the [`common`](/common/) package.
 
-The [`cmd/build`](/cmd/build/) command line tool described above can be used
-for building and releasing the binaries, and at the same time for generating a
-corresponding provenance file. To use this tool, the developers need to provide
```

Comment:
"the developers" here refers to the developers of the project who want to build and release their libraries/binaries using transparent-release, not the transparent-release development team :)